dotnet/api/Infrastructure/Data.SqlServer/Migrations/20200603183650_InitialCreate.cs

```diff
@@ -9,7 +9,8 @@ public partial class InitialCreate : GBFlattenedMigration
     {
         protected override void Up(MigrationBuilder migrationBuilder)
         {
-            if (!ValidateFlattenedShouldRun(InitialMigrationId))
+            var environmentIsTest = Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT") != "Testing";
```

Comment:
I saw this environment variable somewhere else - do we know of a constant that .NET provides for either of these? If not, might be worth breaking out into a constant somewhere...

Edit: Just found `Testing`: https://github.com/AndcultureCode/AndcultureCode.CSharp.Core/blob/main/src/AndcultureCode.CSharp.Core/Constants/EnvironmentConstants.cs

Reply:
Oh nice! Didn't realize there was a constant. Makes things easy.